fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/FromDays.java

```diff
@@ -38,6 +39,7 @@ public class FromDays extends ScalarFunction
         implements UnaryExpression, ExplicitlyCastableSignature, AlwaysNullable {
 
     public static final List<FunctionSignature> SIGNATURES = ImmutableList.of(
+            FunctionSignature.ret(DateV2Type.INSTANCE).args(IntegerType.INSTANCE),
             FunctionSignature.ret(DateType.INSTANCE).args(IntegerType.INSTANCE)
```

Comment:
should not have two signature with same input. remove sig with date return type. and update from_days constant folding in `org.apache.doris.nereids.trees.expressions.functions.executable.DateTimeExtractAndTransform`